.github/workflows/matrix.yml

```diff
@@ -230,7 +230,7 @@ jobs:
       if: ${{ env.SKIP == 'false' && runner.os == 'Linux' }}
       run: |
           sudo apt-get update
-          sudo apt-get install libncursesw5-dev ccache gettext parallel
+          sudo apt-get install libncursesw5-dev ccache gettext parallel python-is-python3
```

Comment:
This works for CI, but I'm not convinced we can require our users to have this installed. Even if it works for ubuntu, it might not for other distros.

I think it would be much better to handle this in Makefile instead, since we can. Something like 
```Makefile
PYTHON=$(shell which python || which python3)
version:
    $(PYTHON) build-scripts/version.py
```